### ISSUE TITLE
[BUG] Cannot create a new imap account

### ISSUE THREAD

**Issue by chongma** [docker-mailserver/docker-mailserver#2793]
### Miscellaneous first checks

- [X] I checked that all ports are open and not blocked by my ISP / hosting provider.
- [X] I know that SSL errors are likely the result of a wrong setup on the user side and not caused by DMS itself. I'm confident my setup is correct.

### Affected Component(s)

dovecot doesn't allow login with newly created account

### What happened and when does this occur?

```Markdown
Running `docker exec mail setup email add username@domain.tld` results in repeatedly outputting `[   INF   ]  Waiting for dovecot to create '/var/mail/domain.tld/username/'`

Logging in through IMAP results in

Sep 24 13:49:23 localhost dovecot: imap-login: Login: user=<username@domain.tld>, method=PLAIN, rip=81.133.251.237, lip=172.23.0.2, mpid=16939, TLS, session=<oqdRuWvplK9Rhfvt>
Sep 24 13:49:23 localhost dovecot: imap(username@domain.tld)<16939><oqdRuWvplK9Rhfvt>: Logged out in=75 out=640 deleted=0 expunged=0 trashed=0 hdr_count=0 hdr_bytes=0 body_count=0 body_bytes=0
```

Running `ls /home2/mail/maildata/domain.tld/username -al` results in
```
total 16
drwxr-xr-x.   2 root root  4096 Sep 24 13:52 .
drwx------. 190 5000 5000 12288 Sep 24 13:52 ..
``` 

Running `docker exec mail setup email del username@domain.tld` results in
```
[   INF   ]  'username@domain.tld' and potential aliases deleted
[  ERROR  ]  No difference after call to 'sed' in 'sedfile' (sed -i -e /^username@domain\tld:.*$/d /tmp/docker-mailserver/dovecot-quotas.cf)
[ WARNING ]  Quota for 'username@domain.tld' couldn't be deleted in '/tmp/docker-mailserver/dovecot-quotas.cf'
[   INF   ]  Mailbox deleted
```
```


### What did you expect to happen?

```Markdown
I expected a new account to be created
```

### How do we replicate the issue?

```Markdown
1.
2.
3.
...
```

### DMS version

v11.1.0

### What operating system is DMS running on?

Linux

### What instruction set architecture is DMS running on?

x86_64 / AMD64

### What container orchestration tool are you using?

Docker Compose

### docker-compose.yml

```yml
version: '3.8'

services:
  mailserver:
    image: docker.io/mailserver/docker-mailserver:latest
    hostname: ${HOSTNAME}
    domainname: ${DOMAINNAME}
    container_name: ${CONTAINER_NAME}
    env_file: mailserver.env
    # To avoid conflicts with yaml base-60 float, DO NOT remove the quotation marks.
    ports:
      - "25:25"
      - "143:143"
      - "465:465"
      - "587:587"
      - "993:993"
      - "995:995"
      - "4190:4190"
    volumes:
      - /home2/mail/maildata:/var/mail
      - /home2/mail/mailstate:/var/mail-state
      - /home2/mail/maillogs:/var/log/mail
      - /etc/localtime:/etc/localtime:ro
      - ./config/:/tmp/docker-mailserver/${SELINUX_LABEL}
      #- ./config/dovecot/fts-xapian-plugin.conf:/etc/dovecot/conf.d/10-plugin.conf:ro
      - ./cron/sa-learn:/etc/cron.d/sa-learn
      #- ./docker-data/dms/cron/fts_xapian:/etc/cron.d/fts_xapian
      - /etc/letsencrypt:/etc/letsencrypt:ro
    restart: always
    stop_grace_period: 1m
    cap_add: [ "NET_ADMIN", "SYS_PTRACE" ]

#volumes:
#  maildata:
#  mailstate:
#  maillogs:
```

### What level of experience do you have with Docker and mail servers?

- [ ] I am inexperienced with docker
- [ ] I am inexperienced with mail servers
- [ ] I am uncomfortable with the CLI

### Code of conduct

- [X] I have read this project's [Code of Conduct](https://github.com/docker-mailserver/docker-mailserver/blob/master/CODE_OF_CONDUCT.md) and I agree
- [X] I have read the [README](https://github.com/docker-mailserver/docker-mailserver/blob/master/README.md) and the [documentation](https://docker-mailserver.github.io/docker-mailserver/edge/) and I searched the [issue tracker](https://github.com/docker-mailserver/docker-mailserver/issues?q=is%3Aissue) but could not find a solution

**Comment by chongma** [comment]
Sorry, it was a roundcube issue not docker mailserver.
$config['auto_create_user'] = true;
was set to false

**Comment by nekrondev** [comment]
Well... unfortunately this must be a bug. I tried to create a test mailaccount on my server just today and went into the same endless loop of "Waiting for dovecot to create..." messages. Eventually after many minutes I gave up and stopped the script. In the end no new account was created.

**Comment by casperklein** [comment]
Please open a new issue and provide the necessary information.
